ERCS/erc-7682.md

```diff
@@ -0,0 +1,79 @@
+---
+eip: 7682
+title: Auxiliary Funds Capability
+description: A way for wallets to indicate to apps that they have access to additional funds
+author: Lukas Rosario (@lukasrosario), Wilson Cusack (@wilsoncusack)
+discussions-to: https://ethereum-magicians.org/t/erc-7682-auxiliary-funds-capability/19599
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-09
+requires: 5792
+---
+
+## Abstract
+
+An [EIP-5792](./eip-5792.md) compliant capability that allows wallets to indicate to apps that they have access to funds beyond those that can be accounted for by looking up balances onchain given the wallet's address.
```

Comment:
It'd be nice to have a bit more technical detail in the abstract. How do wallets indicate this? Where could these funds come from?